Classes/Controller/Cart/ActionController.php

```diff
@@ -52,15 +52,7 @@ public function injectPaymentMethodsService(PaymentMethodsServiceInterface $paym
         $this->paymentMethodsService = $paymentMethodsService;
     }
 
-    public function injectShippingMethodsService(ShippingMethodsServiceInterface $shippingMethodsService): void
-    {
-        $this->shippingMethodsService = $shippingMethodsService;
-    }
-
-    public function injectSpecialOptionsService(SpecialOptionsServiceInterface $specialOptionsService): void
-    {
-        $this->specialOptionsService = $specialOptionsService;
-    }
```

Comment:
Why are `specialOptions` no longer injected here?

Reply:
I have probably not restored everything here that rector had changed. I fixed it with 1b75dec.

Reply:
Great!